File: app/src/main/java/joebruckner/lastpick/widgets/OnAnimationFinishedListener.java
package joebruckner.lastpick.widgets;

import android.view.animation.Animation;

public abstract class OnAnimationFinishedListener implements Animation.AnimationListener {

	@Override public void onAnimationStart(Animation animation) {}

	@Override public abstract void onAnimationEnd(Animation animation);

	@Override public void onAnimationRepeat(Animation animation) {}
}

File: app/src/main/java/joebruckner/lastpick/widgets/OnAnimationFinishedListener.kt
package joebruckner.lastpick.widgets

import android.view.animation.Animation

abstract class OnAnimationFinishedListener : Animation.AnimationListener {

    override fun onAnimationStart(animation: Animation) {
    }

    abstract override fun onAnimationEnd(animation: Animation)

    override fun onAnimationRepeat(animation: Animation) {
    }
}
